interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the software on the local machine that tracks changes to whatever coding you do to a file. The difference between git and Github is that git is on your local machine while Github is a cloud based software based in Idaho that shows what changes were made.
 
-Researched answer:
+Researched answer: Git is an open sourced, cloud based software that is used for sharing and working collaboratively on the same code base. Git is accessed on the command line on your computer. Github is a code hosting platform for version control and collaboration that uses Git.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The JavaScript operators that will return a boolean value are logical operators, relational operators, and equality operators.
 
-Researched answer:
+Researched answer: Boolean expressions in JavaScript return true or false. JavaScript operators that will return a boolean value are equality operators, relational operators, and logical operators.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is the location of data in either a string or an array. The difference between an index and a value is that an index is the location of the value while the value is what represents the data or information.
 
-Researched answer:
+Researched answer: An index contains the position of a regular expression that matches within a string. A value is the content of each element in an array that must be a JavaScript data type.
```

Comment:
I am curious to hear more about what you mean by regular expression in this context.